### DISCUSSION TITLE
C/2021 A1 Comet Leonard

### DISCUSSION THREAD

**Discussion by JChase67** [Stellarium/stellarium#2020] [Ideas]
Comet Leonard is promising to be a great sight during November and December.  Can we please get a quick update to add it to the objects database?

**Comment by gzotti** [comment]
Use the Solar System Editor plugin to update elements.

**Comment by alphanimal** [comment]
Awesome! Looking forward to seeing another comet! I just imported it myself so here's how I did it:
![image](https://user-images.githubusercontent.com/29342016/141176694-900a48b3-60a3-46b7-9b7c-e0b55e7d7b39.png)

**Comment by Sempervivum** [comment]
@alphanimal That's brilliant, thanks for these instructions!
